### ISSUE TITLE
Breadcrumb dark mode accessibility issue

### ISSUE THREAD

**Issue by RhysMAdey** [DiamondLightSource/sci-react-ui#110]
Breadcrumb in dark mode fails accessibility for colour threshold

https://coolors.co/contrast-checker/ffffff-2641a2
